host/libvirt_lxc_backend.go

```diff
@@ -669,11 +672,28 @@ func (c *libvirtContainer) watch(ready chan<- error) error {
 			g.Log(grohl.Data{"at": "get_streams", "status": "error", "err": err.Error()})
 			return err
 		}
+
 		log := c.l.openLog(c.job.ID)
 		defer log.Close()
-		// TODO: log errors from these
-		go log.Follow(1, stdout)
-		go log.Follow(2, stderr)
+
+		muxConfig := logmux.Config{
+			AppName: c.job.Metadata["flynn-controller.app"],
```

Comment:
`ID` was initially used because the initlog was being sent too, and it's not an FD. I kept it b/c i figured that in the future we may collect from some other source than STDOUT/STDERR (e.g. `/dev/log`). The underlying FD number is not constant like with STDOUT/STDERR, so `ID` (which the example prefix used in the RFC) avoids the possibly faulty assumption that this number value is the FD number the process sees.